projects/packages/sync/src/modules/class-posts.php

```diff
@@ -53,6 +53,16 @@ class Posts extends Module {
 	 */
 	private $import_end = false;
 
+	/**
+	 * Max bytes allowed for post_content => length.
+	 * Current Setting : 5MB.
```

Comment:
Current setting looks like it's 2.5 MB (using the 1000-based definition), not 5 MB. Or else the value isn't actually bytes.

Reply:
You are correct, went back through my notes and not sure where that disconnect came in.